docs/guide/plugins.md

```diff
@@ -0,0 +1,38 @@
+# Plugins
+
+Plugins are now supported in the application.
+
+In order to support these, we need:
+
+1. Build docker with the plugin
```

Comment:
I would remove this, in theory you could use them with all the others ways supported, you don't need docker